tests/testthat/test-lsbinomialestimation.R

```diff
@@ -0,0 +1,1561 @@
+context("Learn Bayes - Binomial Estimation")
+
+skip_on_travis()
```

Comment:
Guess these skips can go?

You're missing a .travis.yml file in the root of the repo -- if you add that the tests should start

Reply:
yes, thanks! Completely missed them.